cdhweb/static_src/global/components/image.scss

```diff
@@ -0,0 +1,58 @@
+// .sk-image {
+// }
+
+.sk-image--small {
+  :where(img) {
+    inline-size: 50%;
```

Comment:
there's a mix of logical and physical properties on this project, right? How do you decide when to use `width` and when to use `inline-size`?

Reply:
my system is "when i remember" 😬